client_tools.py

```diff
@@ -54,7 +54,8 @@ def parse_command_line():
 
 
 def main(args):
-
+    # clearing import logs
```

Comment:
clearing text from batch report .txt at the beginning of each run of client tools.